src/FSharpPlus/Data/NonEmptyList.fs

```diff
@@ -185,7 +185,7 @@ type NonEmptyList<'t> with
     static member Map (x: NonEmptyList<'a>, f: 'a->'b) = NonEmptyList.map f x
     
     [<EditorBrowsable(EditorBrowsableState.Never)>]
-    static member MapIndexed (x: NonEmptyList<_>, f) = NonEmptyList.mapi f x
+    static member MapIndexed (x: NonEmptyList<_>, f, [<Optional>]_impl: MapIndexed) = NonEmptyList.mapi f x
```

Comment:
Possible solutions
1- 

```
#if FABLE_COMPILER
    static member MapIndexed (x: NonEmptyList<_>, f) = NonEmptyList.mapi f x
#else
    static member MapIndexed (x: NonEmptyList<_>, f, [<Optional>]_impl: MapIndexed) = NonEmptyList.mapi f x#endif

```

2- 

```
#if !FABLE_COMPILER
    static member MapIndexed (x: NonEmptyList<_>, f, [<Optional>]_impl: MapIndexed) = NonEmptyList.mapi f x
#endif
```

any other solution ? please suggest what to do next

Reply:
I've reverted the addition of the optional parameter